cloudrun-malware-scanner/cloudbuild.yaml

```diff
@@ -0,0 +1,25 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/clamav-scanner:latest', '.']
```

Comment:
instead of creating a new "latest" build everytime

i think we want to:
1. generate new build with unique id
2. tag the new build with "latest"

Reply:
What's the difference between these 2 approaches? It seems it doesn't save an extra image.

Reply:
Done.